packages/core/src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as classNames from "classnames";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IProps } from "../../common/props";
+
+// allow the empty interface so we can label it clearly in the docs
+// tslint:disable-next-line:no-empty-interface
+export interface INavbarProps extends IProps {
+    // Empty
+}
+
+export const Navbar: React.SFC<INavbarProps & { children?: React.ReactNode }> = ({ children, className }) => (
```

Comment:
🤔 we explicitly don't write SFCs in Blueprint because they can't be `ref`'ed. I know it's obnoxious but let's write full components.

Reply:
separate task: would be awesome to re-export the `Navbar*` components here as `Navbar.*`: `Navbar.Group`, `Navbar.Divider`, etc.

so you can either import the separate components or just the one root `Navbar` and have access to the others. [see `Tabs2` for inspiration](https://github.com/palantir/blueprint/blob/master/packages/core/src/components/tabs2/tabs2.tsx#L90-L92).